src/CommonDropdown.php

```diff
@@ -216,7 +216,7 @@ public static function displayCentralHeader(
         ?string $title = null,
         ?array $menus = null
     ): void {
-        if (is_null($menus)) {
+        if (empty($menus)) {
```

Comment:
does `empty()` also handle `null` cases? since it's still the default value; it has to be handled

Reply:
Yes it does: 

![image](https://user-images.githubusercontent.com/42734840/187196140-c63cf11a-2a89-40b7-823d-cc06e77357a6.png)

Reply:
:+1: